test/src/main/java/org/apache/accumulo/test/functional/CompactionIT.java

```diff
@@ -68,6 +116,54 @@ protected int defaultTimeoutSeconds() {
     return 4 * 60;
   }
 
+  @Test
+  public void testBadSelector() throws Exception {
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+      final String tableName = getUniqueNames(1)[0];
+      c.tableOperations().create(tableName);
+      // Ensure compactions don't kick off
+      c.tableOperations().setProperty(tableName, Property.TABLE_MAJC_RATIO.getKey(), "10.0");
```

Comment:
setting these props are eventually consistent, so not sure when tserver will see the config update.  Could set the prop at table creation time instead, avoiding any possible race conditions in the test.